models/repository/repository.go

```diff
@@ -15,14 +20,24 @@ type RepositoryStruct struct {
 	RepositoryID int64
 	Owner        sql.NullString
 	Name         sql.NullString
+	WebhookKey   string
 	database     db.DB
 }
 
-func NewRepository(id int64, repositoryID int64, owner string, name string) *RepositoryStruct {
+func GenerateWebhookKey(seed string) string {
+	h := md5.New()
+	io.WriteString(h, strconv.FormatInt(time.Now().Unix(), 10))
+	io.WriteString(h, seed)
+	token := fmt.Sprintf("%x", h.Sum(nil))
+
+	return token
+}
+
+func NewRepository(id int64, repositoryID int64, owner string, name string, webhookKey string) *RepositoryStruct {
```

Comment:
__[golint]__
* [golint] exported function NewRepository should have comment or be unexported